pyproject.toml

```diff
@@ -29,6 +29,7 @@ PyYAML = "^5"
 requests = "^2"
 structlog = ">=19"
 tenacity = ">=5"
+pydantic = "^1.8.1"
```

Comment:
Given this is a library, we might want to consider relaxing this requirement some. We have moved the other dependencies to just major version requirements. Getting too specific causes a lot of dependency issues with consuming projects.

I'm thinking something like `>=1` or `^1`.